Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift

```diff
@@ -16,14 +16,18 @@ extension String {
 }
 
 class MainTabBarController: UITabBarController {
-    
-    
+
+    private let localNotificationManager: LocalNotificationInterface = LocalNotificationManager()
     private let blurEffectIntensity = 0.3
     
     override func viewDidLoad() {
         super.viewDidLoad()
         configureTabBar()
         setupViewControllers()
+
+        // Local Notifications
+        localNotificationManager.removeAllNotifications()
```

Comment:
In 20 row you just created an instance
What is the main purpose to remove?


Let's move it to deinit {}

Reply:
The main purpose of deletion is to protect against excessive addition of notifications.
deinit{} is not called by MainTabBarController and moving the deletion call to deinit{} is considered impractical.

Reply:
"excessive addition of notifications" - in what cases it should be happened?

Reply:
In cases someone changes the time setting for notifications in the code in the future.

Reply:
How "changes the time setting for notifications" relates with subscribe/remove on notifications?

Reply:
I mistakenly thought that when changing the notification sending time settings, additional notifications would be created, because this happened earlier when I used the notification ID generated by the UUID. After I switched to using the identifier as a constant, the problem disappeared. The call to delete notifications can be removed.